src/gwt/src/org/rstudio/studio/client/workbench/views/packages/PackagesPane.java

```diff
@@ -199,7 +221,24 @@ protected Toolbar createMainToolbar()
       
       // update packages
       toolbar.addLeftWidget(commands_.updatePackages().createToolbarButton());
-      projectButtonSeparator_ = toolbar.addLeftSeparator();
+      toolbar.addLeftSeparator();
+
+      // manage repositories
+      ToolbarPopupMenu repositoryMenu = new ToolbarPopupMenu();
+      repositoryMenu.addItem(new MenuItem("Use CRAN Repository", false, (ScheduledCommand) null));
```

Comment:
I think most of this makes sense to me. I'm not quite following this bit? Are these hardcoded for now but will be based on the list of what I pass into the IDE later?

Reply:
Yeah, that's right -- perhaps I should remove this from the PR, and instead do this separately with a more fully-fledged implementation.